content/02.main-text.md

```diff
@@ -1,6 +1,6 @@
 ## Introduction {.page_break_before}
 
-Openness in research --- which includes sharing code, data, and manuscripts --- benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers, and the public.
+Openness in research --- which includes sharing code, data, and manuscripts --- benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers [@doi:10.1101/344804], and the public.
```

Comment:
I'm in favor of removing the reference to "Developing a modern data workflow for living data" here. It's a bizarrely specific reference when we don't cite other more general works. We probably should cite more general works, but these points are also rather self evident.

Reply:
@dhimmel despite what I wrote above, upon further consideration I agree with you about removing it here.  It fits better in the continuous integration paragraph.  Are you okay adding it in that paragraph?

Reply:
Yeah I think it is appropriate for the CI sentence... It's a similar use case of CI